packages/hint/tests/lib/fixtures/browserslist-multiple/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "app",
+    "version": "1.0.0",
+    "browserslist": ["chrome 45"]
+  }
+  
```

Comment:
```suggestion
```

Reply:
@antross requested for a newline at the end of the test fixtures... https://github.com/webhintio/hint/pull/1599#discussion_r244851401 
I looked a some other fixtures... can't seem to discern what the standard is, some have newlines, some don't. Should I keep or remove?

Reply:
> requested for a newline at the end of the test fixtures

Sorry the misunderstanding, my comment was about removing the extra empty line at the end.

>  can't seem to discern what the standard is, some have newlines, some don't. Should I keep or remove?

[newlines should be kept](https://github.com/webhintio/hint/blob/5ddaa75e4a2634b28c4b364ed127c38906896011/.editorconfig#L15).